### PR TITLE
Identity  Changelog + README update May 2020

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 1.1.0-beta.4 (Unreleased)
+## 1.1.0-beta.4 (2020-05-06)
+- Added `IntelliJCredential` support to `DefaultAzureCredential`.
+- Added `VsCodeCredential` support to `DefaultAzureCredential`.
+- Added  support to disable specific credentials `DefaultAzureCredential` authentication flow.
+
```
Comment (g2vinay) [inline sdk/identity/azure-identity/CHANGELOG.md:7]: @jianghaolu 
What specifically do we need to add for Persistent token cache support here ?

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 1.1.0-beta.4 (Unreleased)
+## 1.1.0-beta.4 (2020-05-06)
```
Comment (alzimmermsft) [inline sdk/identity/azure-identity/CHANGELOG.md:3]: Mind adding in the missing patch release history for 1.0.4, 1.0.5, and 1.0.6 below 1.1.0-beta.1

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,8 @@
 - Added `tokenRefreshOffset` set method in all the credential builders.
 - Added `httpClient` set method in all the credential builders.
 - Updated `DefaultAzureCredential` to enable authenticating through the Azure CLI
+- Added shared token cache support for MacOS Keychain, Gnome Keyring, and plain text for other Linux environments
+- Added option to write to shared token cache from `InteractiveBrowserCredential`, `AuthorizationCodeCredential`, `UsernamePasswordCredential`, and `DeviceCodeCredential`
```
Comment (schaabs) [inline sdk/identity/azure-identity/CHANGELOG.md:27]: shouldn't these changes be under 1.1.0-beta.4 not 1.1.0-beta.2?

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 2 above)
Comment (g2vinay) [reply sdk/identity/azure-identity/CHANGELOG.md:3]: Shouldn't this be ordered via the date they released ?

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 3 above)
Comment (g2vinay) [reply sdk/identity/azure-identity/CHANGELOG.md:27]: Yep, it was accidentally pushed here, I'll update it.

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 2 above)
Comment (alzimmermsft) [reply sdk/identity/azure-identity/CHANGELOG.md:3]: By date would be best

**Pull request merged**